web/src/app/shared/models/notifications.ts

```diff
@@ -310,5 +310,9 @@ export const notifications = new Map<NOTIFICATION_TYPES, Pair>([
   [NOTIFICATION_TYPES.INFO_ALERT_2, new Pair(
     '',
     'The selected Workbasket is empty!'
+  )],
+  [NOTIFICATION_TYPES.WARNING_CANT_COPY, new Pair(
+    '',
+    'Can\'t copy an unsaved classification!'
```

Comment:
This warning is misleading in my opinion, the problem is not, that the classification has not been saved, but instead, that it has not been created.